config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`config` to configure the app on which theme to use. `theme` here depicts not only the CSS, but the set of components to be used.